highlight/highlight
# Tune logs virtual table

## Summary
Reports of the logs table flickering which is making it hard to use. Make the following updates:
- load 50 extra rows (i.e. 1 page) into the virtual table
- do not use debug mode
- add `data-index` to resolve warning
- memoize rows to allow expanding data objects

## How did you test this change?
1) Visit the logs table and confirm better UX:
- scroll down to load more
- scroll up
- open rows
- expand objects

## Are there any deployment considerations?
N/A

## Does this work require review from our design team?
N/A
